packages/chain-adapters/src/solana/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const SOLANA_COMPUTE_UNITS_BUFFER_MULTIPLIER = 1.4
+
+export const SOLANA_MINIMUM_INSTRUCTIONS_NUMBER = 3
```

Comment:
```suggestion
export const SOLANA_MINIMUM_INSTRUCTIONS = 3
```

or 

```suggestion
export const SOLANA_MINIMUM_INSTRUCTION_COUNT = 3
```

Reply:
Updated in [275f332](https://github.com/shapeshift/web/pull/8327/commits/275f332ecfa01fe8a67b10465016c05190d26415)